packages/core/src/schema/lavamoat-policy.v0-0-1.schema.ts

```diff
@@ -65,9 +65,26 @@ export interface NodeLocation {
  * @deprecated - Use `true` instead
  */
 export type GlobalPolicyRead = 'read'
+
+/**
+ * A writable global property
+ */
 export type GlobalPolicyWrite = 'write'
 
-export type GlobalPolicyValue = GlobalPolicyRead | GlobalPolicyWrite | boolean
+/**
+ * A property which can be defined via {@link Object.defineProperty} and/or
+ * {@link Object.defineProperties}
+ */
+export type GlobalPolicyRedefine = 'redefine'
+
+/**
+ * Possible values for global policy
+ */
+export type GlobalPolicyValue =
+  | boolean
+  | GlobalPolicyRead
```

Comment:
```suggestion
  | GlobalPolicyRead // deprecated
```

or

```ts
export type GlobalPolicyValue =
  | boolean
  | GlobalPolicyRedefine
  | GlobalPolicyWrite
  // deprecated
  | GlobalPolicyRead
```

(both endo style)

edit: nevermind, endo is js not ts 🤦‍♂️

Reply:
or

https://github.com/LavaMoat/LavaMoat/blob/82e02e06128edab69fdb0581574375825340b6e1/packages/core/src/schema/lavamoat-policy.v0-0-1.schema.ts#L65

enough

since code editors strikeout these days